src/sentry/search/events/datasets/metrics.py

```diff
@@ -150,6 +159,7 @@ def function_converter(self) -> Mapping[str, fields.MetricsFunction]:
                     snql_distribution=lambda args, alias: self._resolve_percentile(
                         args, alias, 0.5
                     ),
+                    result_type_fn=self.reflective_metric_type(),
```

Comment:
nit: argument ends with `_fn` but we pass value, not a function.

Reply:
Sorry, not sure what you mean, `reflective_metric_type` returns a callable?

Reply:
so i expect here -> `result_type_fn=self.reflective_metric_type`

Reply:
never mind, i miss 1 line.